quantum/quantum.c

```diff
@@ -354,7 +354,12 @@ bool process_record_quantum(keyrecord_t *record) {
 #    else
             if (!record->event.pressed) {
 #    endif
-                rgblight_increase_hue();
+                uint8_t shifted = get_mods() & (MOD_BIT(KC_LSHIFT) | MOD_BIT(KC_RSHIFT));
+                if (shifted) {
```

Comment:
There's no need for a variable here, just do `if (get_mods() & MOD_MASK_SHIFT) { ... }` directly :)

Reply:
Thanks.  Fixed now, including preexisting occurrences.